examples/wiki/wiki.py

```diff
@@ -9,6 +9,10 @@
 WIKIPART = re.compile(r'([A-Z][a-z0-9_]+)')
 WIKIWORD = re.compile(r'([A-Z][a-z0-9_]+(?:[A-Z][a-z0-9_]+)+)')
 
+@app.route('/', methods=['GET'])
+def redirect():
```

Comment:
this will shadow "redirect" imported from flask module... should probably rename to `homepage_redirect` or something like that (that's what it was called before).